src/FakeItEasy/Core/FakeWrapperConfigurator.cs

```diff
@@ -13,7 +13,10 @@ namespace FakeItEasy.Core
     /// </summary>
     /// <typeparam name="T">The type of fake object generated.</typeparam>
     internal class FakeWrapperConfigurator<T>
-        : FakeOptionsBase<T>, IFakeOptionsForWrappers<T>, IFakeOptionsForWrappers
+        : FakeOptionsBase<T>
+#if FEATURE_SELF_INITIALIZED_FAKES
+        , IFakeOptionsForWrappers<T>, IFakeOptionsForWrappers

```

Comment:
It's a bit weird, and I'm also surprised StyleCop didn't complain. It's OK though.